.github/workflows/wpt-mapper.yml

```diff
@@ -101,3 +101,4 @@ jobs:
 env:
   DEBUG: bidiServer:log
   FORCE_COLOR: 3
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
```

Comment:
`pip` is extremely annoying. It asks to be updated ~3-4 times in every WPT run. Let's just disable these, as we already have `dependabot` in our infra.